src/commands/info/submit.py

```diff
@@ -75,22 +70,32 @@ async def on_reaction_add(self, reaction: discord.Reaction, user: discord.User):
         submission = self.SUBMISSION_REGEX.match(reaction.message.content)
         if not submission:
             return
+        group = submission.groupdict()
         
         # If the user doesn't have the verifier role, remove the reaction
         # If the verifier role is not set, then anyone can verify the score
         if VERIFIER_ROLE and VERIFIER_ROLE not in [role.id for role in user.roles]:
             await reaction.message.remove_reaction(reaction.emoji, user)
             return
         
-        # TODO: Add logic to handle adding the score to the database
         if reaction.emoji == "✅":
-            pass
+            # Add the score to the database
+            with get_db() as db:
+                category = db.query(LeaderboardCategory).filter(LeaderboardCategory.category_name == group['category']).first()
+                score = LeaderboardScore(msg_id=reaction.message.id, user_id=group['user_id'], score=group['score'], url=group['url'], category_id=category.category_id)
+                db.add(score)
+                db.commit()
```

Comment:
Turn this into a CRUD method inside of `sql.models.leaderboard` or maybe `sql.crud.leaderboard`